src/api/spec/factories/configuration.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :configuration do
```

Comment:
Yeah, but it's a convenient factory to get easily the attributes for updating a configuration!